packages/clerk-js/src/ui/components/OrganizationProfile/VerifiedDomainPage.tsx

```diff
@@ -104,6 +163,24 @@ export const VerifiedDomainPage = withCardStateProvider(() => {
     domain: domain?.name,
   });
 
+  const calloutLabel = useCalloutLabel(domain, {
+    prefix: localizationKeys(
+      `organizationProfile.verifiedDomainPage.enrollmentTab.calloutLabels.${
+        (domain?.totalPendingInvitations || 0) > 1 ? 'prefix_many' : 'prefix_single'
```

Comment:
See my comment on the localization file, duplicating the sentence might be worth it to avoid the extra complexity here.